app/_how-tos/get-started-with-metering-and-billing.md

```diff
@@ -0,0 +1,253 @@
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+
+min_version:
+    gateway: '3.4'
+next_steps:
+  - text: See all {{site.base_gateway}} tutorials
+    url: /how-to/?products=gateway
+  - text: Learn about {{site.base_gateway}} entities
+    url: /gateway/entities/
+  - text: Learn about how {{site.base_gateway}} is configured
+    url: /gateway/configuration/
+  - text: See all {{site.base_gateway}} plugins
+    url: /plugins/
+automated_tests: false
+---
+
+## Create a Consumer
+
+Create the two consumers, one for the free and one for premium. These will be mapped to our customers/subjects later on
+
+[Consumers](/gateway/entities/consumer/) let you identify the client that's interacting with {{site.base_gateway}}.
+We're going to use key [authentication](/gateway/authentication/) in this tutorial, so the Consumer needs an API key to access any {{site.base_gateway}} Services.
+
+<!--vale off-->
+{% entity_examples %}
+entities:
+  consumers:
+    - username: kong-air
+      keyauth_credentials:
+        - key: air-key
+    - username: kong-travel
+      keyauth_credentials:
+        - key: travel-key
+{% endentity_examples %}
+<!--vale on-->
+
+## Enable authentication
+
+Authentication lets you identify a Consumer so that you can apply rate limiting.
+This example uses the [Key Authentication](/plugins/key-auth/) plugin, but you can use any [authentication plugin](/plugins/?category=authentication) that you prefer.
+
+Enable the plugin globally, which means it applies to all {{site.base_gateway}} Services and Routes:
+
+<!--vale off-->
+{% entity_examples %}
+entities:
+  plugins:
+    - name: key-auth
+      config:
+        key_names:
+          - apikey
+{% endentity_examples %}
+<!--vale on-->
+
+## Enable rate limiting
+
+Set rate limits for the consumers. Free consumer gets 1,000 requests for example-service and premium gets 5,000 requests for the same service/api.
+
+Enable the [Rate Limiting plugin](/plugins/rate-limiting/) for the Consumer. 
+In this example, the limit is 5 requests per minute and 1000 requests per hour.
+
+<!--vale off-->
+{% entity_examples %}
+entities:
+  plugins:
+    - name: rate-limiting
+      consumer: kong-air
+      config:
+        month: 1000
+    - name: rate-limiting
+      consumer: kong-travel
+      config:
+        month: 5000
+{% endentity_examples %}
+<!--vale on-->
+
+## Configure ACL plugin
+
+Configure the ACL plugin to only allow access to the service for customers who have signed up, so our two consumers:
+
+<!--vale off-->
+{% entity_examples %}
+entities:
+  consumers:
+    - username: kong-air
+      acls:
+      - group: kong-air
+      keyauth_credentials:
+      - key: air-key
+
+    - username: kong-travel
+      acls:
+      - group: kong-travel
+      keyauth_credentials:
+      - key: travel-key
+{% endentity_examples %}
+<!--vale on-->
+
+^ ChatGPT recommended this to fix my ACL error, tested and it seems to work.
+
+## Create a meter
+
```

Comment:
Should we explain why? Like:
"To meter API request traffic from our consumers we will create a meter:"